spikingjelly/activation_based/cuda_kernel/auto_cuda/base.py

```
@@ -1,18 +1,17 @@
-import numpy as np
 import logging
+import sys
+
+import numpy as np
+import torch
 
 try:
     import cupy
 except BaseException as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `BaseException`

(BLE001)